addons/binding/org.openhab.binding.lutron/src/main/java/org/openhab/binding/lutron/internal/radiora/handler/SwitchHandler.java

```diff
@@ -0,0 +1,82 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.lutron.internal.radiora.handler;
+
+import org.eclipse.smarthome.core.library.types.OnOffType;
+import org.eclipse.smarthome.core.thing.ChannelUID;
+import org.eclipse.smarthome.core.thing.Thing;
+import org.eclipse.smarthome.core.types.Command;
+import org.openhab.binding.lutron.LutronBindingConstants;
+import org.openhab.binding.lutron.internal.radiora.config.SwitchConfig;
+import org.openhab.binding.lutron.internal.radiora.protocol.LocalZoneChangeFeedback;
+import org.openhab.binding.lutron.internal.radiora.protocol.RadioRAFeedback;
+import org.openhab.binding.lutron.internal.radiora.protocol.SetSwitchLevelCommand;
+import org.openhab.binding.lutron.internal.radiora.protocol.ZoneMapFeedback;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Handler for RadioRA switches
+ *
+ * @author Jeff Lauterbach
+ *
+ */
+public class SwitchHandler extends LutronHandler {
+
+    private Logger logger = LoggerFactory.getLogger(SwitchHandler.class);
+
+    private SwitchConfig config;
+
+    public SwitchHandler(Thing thing) {
+        super(thing);
+        this.config = getConfigAs(SwitchConfig.class);
+    }
+
+    @Override
+    public void handleCommand(ChannelUID channelUID, Command command) {
+        if (channelUID.getId().equals(LutronBindingConstants.CHANNEL_SWITCH)) {
```

Comment:
Position literals first in `String` comparisons - that way if the `String` is `null` you won't get a `NullPointerException`, it'll just return `false`. [source: PMD](http://pmd.sourceforge.net/pmd-4.3.0/rules/design.html#PositionLiteralsFirstInComparisons)

Please check all